app/models/project_role.rb

```diff
@@ -5,6 +5,7 @@ class ProjectRole < ActiveHash::Base
   include ActiveModel::Serializers::JSON
 
   self.data = [
+    { id: -1, name: "default" },
```

Comment:
Should we name it `none` to give the explicit idea that any project level role will be revoked? I'm thinking that 'default' might be too ambiguous, may cause confusion around what's the behaviour of the default role.

An alternative would be `viewer`, to make it consistent with the system level roles. We'll also need to update the documentation to reflect this change.